content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,409 @@
+│ │ ╟ b3de55f           │ 05:02 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ c4a46af           │ 05:02 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ daf204c           │ 05:02 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ bdb975c           │ 05:02 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 77dc46c           │ 05:01 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+## Starting from an existing checkpoint
+
+Since you have all of these different checkpoints, you might want to resume
+training from a particular one. For example, maybe your accuracy started
+decreasing at a certain checkpoint and you want to make some changes to fix
+that.
+
+First, we need to apply the checkpoint we want to begin our new experiment from.
+To do that, run the following command:
+
+```bash
+dvc exp apply b3de55f
+```
+
+where _b3de55f_ is the id of the checkpoint you want to reference.
+
+Next, we'll change the learning rate in the _params.yaml_ to `0.001`. Then we'll
+start a new experiment based on an existing checkpoint with the following
+command:
+
+```bash
+dvc exp run
+```
+
+You'll be able to see where the experiment starts from the existing checkpoint
+by running:
+
+```bash
+dvc exp show
+```
+
+You should seem something similar to this in your terminal.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃ step ┃     loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -            │   18 │ 0.12399  │ 0.9625 │ 473987 │ 0.001  │ 0            │
+│ main                  │ Apr 27, 2021 │    - │       -  │      - │ 473987 │ 0.0001 │ 0            │
+│ │ ╓ exp-63287         │ 04:37 PM     │   18 │ 0.12399  │ 0.9625 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 868c144           │ 04:37 PM     │   17 │ 0.12042  │ 0.9616 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 4de4e69           │ 04:37 PM     │   16 │ 0.13808  │ 0.9585 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ d8d633c           │ 04:37 PM     │   15 │ 0.20415  │ 0.9397 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 8fbab93 (b3de55f) │ 04:36 PM     │   14 │ 0.24509  │ 0.9309 │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-4de2a         │ 05:03 PM     │    7 │  0.3096  │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ d50c724           │ 05:03 PM     │    6 │ 0.33246  │ 0.9044 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 29491a9           │ 05:02 PM     │    5 │ 0.36601  │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b3de55f           │ 05:02 PM     │    4 │ 0.41666  │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+```
+
+The existing checkpoint is referenced at the beginning of the new experiment.
```

Comment:
The sha-s would be super helpful here. It might be not clear for the user what is existing and a new mean.

Reply:
What do you mean by the 'sha-s'?

Reply:
@flippedcoder This means not using the `exp-4de2a` names but the Git commit hashes instead. I think if you upgrade to the latest dvc version, you will be able to see both and it will fix this issue.

Reply:
Gotcha. I think I have the latest DVC version. Is it `2.0.18`? I still don't see the Git hashes though. Just the `exp-f923ji` syntax.

Reply:
It's 2.1.0 now. The table format hopefully looks something like:
```
┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━┓
┃ Experiment              ┃ Created      ┃ step ┃ acc    ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━┩
│ workspace               │ -            │ 1    │ 0.9133 │
│ test                    │ 12:52 PM     │ 1    │ 0.9133 │
│ live                    │ Apr 19, 2021 │ -    │ -      │
│ │ ╓ 7243788 [exp-bffd1] │ 12:46 PM     │ 1    │ 0.9133 │
│ ├─╨ c039cb0             │ 12:46 PM     │ 0    │ 0.8523 │
│ │ ╓ 9121bba [exp-dbb5a] │ 12:46 PM     │ 6    │ 0.9791 │
│ │ ╟ 9790bd2             │ 12:45 PM     │ 5    │ 0.9807 │
│ │ ╟ 5d629a2             │ 12:45 PM     │ 4    │ 0.976  │
│ │ ╟ b94ecb0             │ 12:45 PM     │ 3    │ 0.9714 │
│ │ ╟ 5a7ba19             │ 12:45 PM     │ 2    │ 0.9569 │
│ │ ╟ 0849314             │ 12:45 PM     │ 1    │ 0.9659 │
│ ├─╨ 98c13ed             │ 12:44 PM     │ 0    │ 0.95   │
└─────────────────────────┴──────────────┴──────┴────────┘
```